### ISSUE TITLE
AttributeError: 'pywr._core.Timestep' object has no attribute '_index'

### ISSUE THREAD

**Issue by Jmiguel17** [pywr/pywr-dcopf#5]
Hi @jetuk,

I was running an MOEA search and I got this error:

```
Simplex solve returned: None (0)
Simplex status: solution is unbounded (6)
Scenario ID: 0

  File "c:\users\n65847jg.ds\dropbox (the university of manchester)\doctorado\pywr-dcopf\pywr_dcopf\__init__.py", line 21, in solve
    return self._cy_solver.solve(model)
  File "pywr_dcopf\_glpk_dcopf_solver.pyx", line 315, in pywr_dcopf._glpk_dcopf_solver.CythonGLPKDCOPFSolver.solve
  File "pywr_dcopf\_glpk_dcopf_solver.pyx", line 321, in pywr_dcopf._glpk_dcopf_solver.CythonGLPKDCOPFSolver.solve
  File "pywr_dcopf\_glpk_dcopf_solver.pyx", line 460, in pywr_dcopf._glpk_dcopf_solver.CythonGLPKDCOPFSolver._solve_scenario
AttributeError: 'pywr._core.Timestep' object has no attribute '_index'
```
Any idea why I am getting this?

Best,

**Comment by jetuk** [comment]
`pywr._core.Timestep` this was updated in Pywr to change `_index` to `index`. So it should be a quick fix to remove the `_`

**Comment by jetuk** [comment]
I've fixed this in #6. However, it would only have happened if the solver failed to find an optimal solution.

**Comment by Jmiguel17** [comment]
Hi @jetuk, yes, I was investigating why the model failed to find an optimal solution, and I found the problem. However, something weird happened. I ran the optimisation using Platypus and Borg and in both cases, glpk failed to find an optimal solution. However, using Borg, glpk continued "iterating" until finding a solution and the optimisation algorithm continued. In the Platypus case, at the first time glpk failed, the search returned the error you fixed.
